pkg/models/html.go

```diff
@@ -0,0 +1,78 @@
+package models
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"html/template"
+	"io"
+
+	"github.com/cmsgov/easi-app/pkg/sanitization"
+)
+
+// HTML represents html code. It is sanitized to only allow specific tags
+type HTML string
+
+// ToTemplate converts the HTML type to a template.HTML struct
+func (html HTML) ToTemplate() template.HTML {
+	// Note, we sanitize when the data is Unmarshaled. However, if desired it could be sanitized at this entrypoint as well
```

Comment:
I think calling `sanitization.SanitizeHTML()` at this entrypoint as well would be a good idea, since `template.HTML` is supposed to be known-good.